yarn-project/circuit-types/src/logs/l1_payload/l1_note_payload.ts

```diff
@@ -14,10 +14,6 @@ import { Note } from './payload.js';
  */
 export class L1NotePayload {
```

Comment:
Had to do this refactor as the payload now contains the private and public values separately. This resulted in a ton of changes around the TS codebase.